console/src/workspace/ontology.tsx

```diff
@@ -115,9 +114,8 @@ const TreeContextMenu: Ontology.TreeContextMenu = (props): ReactElement => {
     void Group.fromSelection(props);
   };
 
-  const clusterKey = Cluster.useSelectActiveKey();
   const handleLink = (): void => {
-    const toCopy = `synnax://cluster/${clusterKey}/workspace/${selection.resources[0].id.key}`;
+    const toCopy = `synnax://cluster/${client.key}/workspace/${selection.resources[0].id.key}`;
```

Comment:
This won't work. The client key is not equal to the cluster key. Use the cluster key instead.